test/controllers/project_skill_controller_test.exs

```diff
@@ -1,14 +1,16 @@
 defmodule CodeCorps.ProjectSkillControllerTest do
-  use CodeCorps.ApiCase
+  use CodeCorps.ApiCase, resource_name: :project_skill
 
   alias CodeCorps.ProjectSkill
   alias CodeCorps.Repo
 
-  defp build_payload, do: %{ "data" => %{"type" => "project-skill", "attributes" => %{}}}
+  @valid_attrs %{}
```

Comment:
Ok, we actually tried that and noticed the test would pass.  But also noticed in several other tests that `@valid_attrs` and `@invalid_attrs` were set equal to `#{}` so wanted to check before changing it. In some cases, these weren't referenced anywhere else in the tests besides where they were "assigned".  Are these tests that haven't been refactored? Or partially refactored?